### PR TITLE
CQLHS-76 | Retrieve project & display on Project Evaluations page

### PR DESCRIPTION
**Pull request by jonathanchaffer**
Sample data on the database is currently missing some values, but on the front end everything is hooked up properly.

- [x] Note: I had to manually edit `api.ts` to add a `client` field to `ProjectOverview`. This PR shouldn't be considered done until the swagger docs are updated to actually include this.

### CODE REVIEW

File: ClientApp/.eslintrc.js [hunk 1]
```diff
@@ -101,7 +101,6 @@ module.exports = {
 
     // React Hooks
     "react-hooks/rules-of-hooks": "error",
-    "react-hooks/exhaustive-deps": "warn",
```
Comment (charkour) [inline ClientApp/.eslintrc.js:104]: Why is this warning removed?

File: ClientApp/src/components/pages/ProjectDirectoryPage/ProjectDirectoryPage.tsx [hunk 2]
```diff
@@ -81,27 +93,45 @@ export const useSortContext = (): {
...
-        title="Project Evaluations"
-      />
-      <SortContext.Provider value={{ context, dispatch }}>
-        <ProjectDirectoryTable
-          projects={filterProjectsByTerm(placeholderData, searchText)}
-        />
-      </SortContext.Provider>
+      {isLoading ? (
+        <Spinner animation="border" />
+      ) : (
```
Comment (charkour) [inline ClientApp/src/components/pages/ProjectDirectoryPage/ProjectDirectoryPage.tsx:111]: This logic is likely going to be repeated on other pages. In the future, some sort of `asyncComponent` component would be helpful to store the loading logic for pages or on-screen widgets.

File: ClientApp/src/components/pages/ProjectDirectoryPage/ProjectDirectoryPage.tsx [hunk 3]
```diff
@@ -81,27 +93,45 @@ export const useSortContext = (): {
...
-        ]}
-        title="Project Evaluations"
-      />
-      <SortContext.Provider value={{ context, dispatch }}>
-        <ProjectDirectoryTable
-          projects={filterProjectsByTerm(placeholderData, searchText)}
-        />
-      </SortContext.Provider>
+      {isLoading ? (
+        <Spinner animation="border" />
```
Comment (charkour) [inline ClientApp/src/components/pages/ProjectDirectoryPage/ProjectDirectoryPage.tsx:110]: Should the spinner be centered?

File: ClientApp/.eslintrc.js (hunk 1 above)
Comment (charkour) [reply ClientApp/.eslintrc.js:104]: Does this relate to the `useEffect()` on line 100 in `ProjectDirectoryPage.tsx`?

Comment (jonathanchaffer) [reply ClientApp/.eslintrc.js]: I removed it for a temporary fix and forgot to add it back in. Thanks for pointing that out!

File: ClientApp/src/components/pages/ProjectDirectoryPage/ProjectDirectoryPage.tsx (hunk 2 above)
Comment (jonathanchaffer) [reply ClientApp/src/components/pages/ProjectDirectoryPage/ProjectDirectoryPage.tsx:111]: Good idea

File: ClientApp/src/components/pages/ProjectDirectoryPage/ProjectDirectoryPage.tsx (hunk 3 above)
Comment (jonathanchaffer) [reply ClientApp/src/components/pages/ProjectDirectoryPage/ProjectDirectoryPage.tsx:110]: Depends on what Ally says. Wouldn't be a difficult change to make

Comment (charkour) [reply ClientApp/src/components/pages/ProjectDirectoryPage/ProjectDirectoryPage.tsx:110]: Sounds good! Thanks. I'm guessing that would fit best in a future ticket.

**Pull request merged**